symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/UserService.java

```diff
@@ -165,6 +191,11 @@ public OboUserService obo(AuthSession oboSession) {
     String usernames = String.join(",", usernameList);
     V2UserList v2UserList = executeAndRetry("searchUserByUsernames",
         () -> usersApi.v3UsersGet(authSession.getSessionToken(), null, null, usernames, true, null));
+
+    if (v2UserList == null || v2UserList.getUsers() == null) {
```

Comment:
as v2UserList is of the same type we could probably factor this logic in a `getUsersOrEmpty(v2UserList)` method